curl/nn/module.py

```diff
@@ -524,7 +525,7 @@ def forward_function(*args, **kwargs):
                         "Please make sure you feed your model CrypTensors when needed.",
                         DeprecationWarning,
                     )
-            elif not self.encrypted:
+            elif not self.encrypted and not self.SUPPORTS_PLAINTEXT_WEIGHTS:
                 if any(isinstance(arg, curl.CrypTensor) for arg in args):
```

Comment:
Is skipping this error here sufficient? Does it work or does it need more changes?

Reply:
Tested it & seems to work indeed.
Can verify by running:
```
from torch import nn
fc = nn.Linear(100, 2)
private_model = curl.nn.from_pytorch(fc, torch.empty((1, 100)), allow_plaintext_weights=True)
private_model.eval()
output_enc = private_model(curl.cryptensor(torch.ones(1,100)))
output_enc.get_plain_text()
```
On a meta note: is this a good time to create testing + CI/CD?

Reply:
> On a meta note: is this a good time to create testing + CI/CD?

Yeah, that'd be nice but we'd have to be careful which tests to run as some take a while.